internal/cmd/asyncapi/account_details.go

```diff
@@ -74,24 +74,36 @@ func (d *accountDetails) getSchemaDetails() error {
 	if err != nil {
 		return err
 	}
-	var unmarshalledSchema map[string]any
+	d.channelDetails.schema = &schema
+	unmarshalledSchema := make(map[string]any)
 	if schema.SchemaType == "" {
-		d.channelDetails.contentType = "application/avro"
-	} else if schema.SchemaType == "JSON" {
+		schema.SchemaType = "AVRO"
+	}
+	switch schema.SchemaType {
+	case "JSON":
 		d.channelDetails.contentType = "application/json"
-	} else if schema.SchemaType == "PROTOBUF" {
-		log.CliLogger.Warn("Protobuf not supported.")
-		d.channelDetails.contentType = "PROTOBUF"
-		return nil
+		break
+	case "AVRO":
+		d.channelDetails.contentType = "application/avro"
+		break
+	case "PROTOBUF":
+		return fmt.Errorf("protobuf")
```

Comment:
Let's come up with a more descriptive error here? If I saw this as a user, I would be really confused.

Reply:
It's a better practice to propagate the user-facing error all the way up the stack!